apps/builtinconf.go

```diff
@@ -39,7 +43,7 @@ var (
 				Receivers: map[string]cg.MetricsReceiver{
 					"hostmetrics": &MetricsReceiverHostmetrics{
 						ConfigComponent:       cg.ConfigComponent{Type: "hostmetrics"},
-						MetricsReceiverShared: cg.MetricsReceiverShared{CollectionInterval: "60s"},
+						MetricsReceiverShared: cg.MetricsReceiverShared{CollectionInterval: time.Second * 60},
```

Comment:
Nit: It reads better if the number comes first.

```suggestion
						MetricsReceiverShared: cg.MetricsReceiverShared{CollectionInterval: 60 * time.Second},
```